contracts/global/PriceFeedStore.sol

```diff
@@ -3,6 +3,174 @@
+    bytes32 public constant override contractType = AP_PRICE_FEED_STORE;
+
+    /// @notice Threshold on number of auditors required to add a price feed
+    uint256 public constant AUDITOR_THRESHOLD = 2;
+
+    //
+    // VARIABLES
+    //
+
+    /// @dev Set of all known price feeds
+    EnumerableSet.AddressSet internal _knownPriceFeeds;
+
+    /// @dev Mapping from token address to its set of allowed price feeds
+    mapping(address => EnumerableSet.AddressSet) internal _allowedPriceFeeds;
+
+    /// @notice Mapping from token address to its equivalent token. A token can use any price feeds of its equivalent.
+    mapping(address => address) public equivalentTokens;
+
+    /// @notice Mapping from price feed address to its data
+    mapping(address => PriceFeedInfo) public priceFeedInfo;
+
+    /// @notice Returns the list of price feeds available for a token
+    function getPriceFeeds(address token) external view returns (address[] memory priceFeeds) {
+        return _allowedPriceFeeds[token].values();
+    }
+
+    /// @notice Returns whether a price feed is allowed to be used for a token
+    function isAllowedPriceFeed(address token, address priceFeed) external view returns (bool) {
+        return _priceFeedVerified(priceFeed)
+            && (
+                _allowedPriceFeeds[equivalentTokens[token]].contains(priceFeed)
+                    || _allowedPriceFeeds[token].contains(priceFeed)
+            );
+    }
+
+    /// @notice Returns the staleness period for a price feed
+    function getStalenessPeriod(address priceFeed) external view returns (uint32) {
+        return priceFeedInfo[priceFeed].stalenessPeriod;
+    }
+
+    function computePriceFeedHash(address priceFeed) public pure returns (bytes32) {
+        return keccak256(abi.encode(priceFeed));
+    }
+
+    /**
+     * @notice Adds a security report for a price feed.
+     * @param priceFeed The price feed for which an audit is added.
+     * @param reportUrl The URL of the security report.
+     * @dev Reverts if the caller is not a registered auditor or if the auditor is forbidden.
+     *      The corresponding access control logic is implemented in AuditManager
+     *      Emits an AuditPriceFeed event upon successful addition of the report.
+     */
+    function addSecurityReport(address priceFeed, string calldata reportUrl) external {
+        bytes32 priceFeedHash = computePriceFeedHash(priceFeed);
+
+        _addSecurityReport(priceFeedHash, msg.sender, reportUrl);
+
+        emit AuditPriceFeed(msg.sender, priceFeed);
+    }
+
+    /**
+     * @notice Adds a new price feed
+     * @param priceFeed The address of the new price feed
+     * @param stalenessPeriod Staleness period of the new price feed
+     * @dev Reverts if the price feed's latest value is not current based on the staleness period
+     */
+    function addPriceFeed(address priceFeed, uint32 stalenessPeriod) external onlyOwner nonZeroAddress(priceFeed) {
```

Comment:
While it makes sense to have one centralized entity to whitelist auditors, having the whole system rely on this entity to add price feeds is a bit too harsh (same applies to bytecode repository).

@0xmikko shall we consider the design when a price feed can be proposed by this entity OR one of auditors and added after a sufficient number of other auditors have committed to it? The question remains who is responsible for whitelisting pairs `(token, priceFeed)` and configuring the staleness period (IMO, both these actions also require confirmation from auditors).